src/annotator/sidebar.ts

```diff
@@ -633,13 +633,13 @@ export class Sidebar implements Destroyable {
    * Shows the sidebar's controls
    */
   show() {
```

Comment:
The difference between `open`/`close` and `show`/`hide` are not immediately obvious from reading the docs. The functions were added in 0bb2f970ddc79df71b04f7b5e1caf11f4bd69a52. The commit message doesn't explain the rationale. The PR https://github.com/hypothesis/client/pull/3061 mentions this was done to preserve the open/closed state after the notebook was dismissed ... but I'm wondering what the original rationale was, before we had looked into screen reader notifications, for hiding the sidebar was in the first place, as opposed to just letting the notebook display above it.

Reply:
Those are good points. I also wondered if we could have re-used open/close, but I guess preserving its state might make sense.

Reply:
> The difference between `open`/`close` and `show`/`hide` are not immediately obvious from reading the docs.

Huh, funny, I don't find this as confusing in terms of what the methods _do_, but I'll admit I don't know _why_ . It appears to me that "open" and "close" do literally that: expand and contract the sidebar and inform anything listening that layout has changed, while "show" and "hide" seem only concerned with the visibility of the sidebar _in toto_ — but more specifically, whether or not the controls to open/close the sidebar are visible. So that seems to be what these methods _do_.

The _why_ part, however, alas, is context I've lost. Potentially it was me following what you might call the _easiest path of first assumption_ about the ideal behavior between the sidebar and the notebook? I certainly wouldn't take anything as inviolable! I can see some comments that I desired the behavior such that the sidebar (layout/open-ness) state would be "restored" after the notebook was closed, but it seems like that might not be necessary if we didn't mess with the open state of the sidebar when the notebook was opened in the first place. However, this probably requires some getting in and messing around and testing what happens. There may be something undesirable about having both open at the same time. But, _mea culpa_, I can't remember what that might be.

It's nice to see we're doing a better job of baking in context over time as we ever improve our software process; it's a pity this context was lost, though.